js/packages.js

```diff
@@ -412,3 +418,66 @@ function updateModal(pkg) {
         document.getElementById('pkg-modal-files').replaceWith(fileDiv)
     }
 }
+
+function clickInstallTab(platform) {
+    let installCode = document.getElementById('install-code')
+    installCode.setAttribute('readonly', false)
+    let windowsTab = document.getElementById('windows-tab')
+    let linuxTab = document.getElementById('linux-tab')
+    switch (platform) {
+        case 'Windows':
+            installCode.textContent =
+                '.\\vcpkg\\vcpkg install ' + selectedPackage.Name
+            windowsTab.classList.add('selected')
+            linuxTab.classList.remove('selected')
+            break
+        case 'Linux':
+            installCode.textContent =
+                './vcpkg/vcpkg install ' + selectedPackage.Name
+            windowsTab.classList.remove('selected')
+            linuxTab.classList.add('selected')
+            break
+        default:
+            console.log('Error: unexpected platform', platform)
+    }
+    installCode.setAttribute('readonly', true)
+}
+
+function copyInstallTab() {
+    let temp = document.getElementById('install-code')
+    temp.value = temp.textContent
+    temp.select()
+    document.execCommand('copy')
+    clearSelection()
+}
+
+// remove the highlight from selected text
+function clearSelection() {
+    if (window.getSelection) {
+        window.getSelection().removeAllRanges()
+    } else if (document.selection) {
+        document.selection.empty()
+    }
+}
+
+// determine what OS the user is on, used to render corresponding package installation code
+function detectOS() {
+    var clientStrings = [
+        { s: 'Windows', r: /(Win)/ },
+        { s: 'Android', r: /Android/ },
+        { s: 'Linux', r: /(Linux|X11(?!.*CrOS))/ },
+        { s: 'iOS', r: /(iPhone|iPad|iPod)/ },
+        { s: 'Mac', r: /Mac/ },
+        { s: 'UNIX', r: /',UNIX/ },
+    ]
+    for (var id in clientStrings) {
+        var cs = clientStrings[id]
+        if (cs.r.test(navigator.platform)) {
+            //assumes of the clients above, only windows and andriod users prefer Windows instructions
+            return cs.s === 'Windows' || cs.s === 'Android'
```

Comment:
This seems weird; I think that this should probably only return `Windows` when the client is using windows, otherwise the unix instructions are fine (especially since the unix instructions often _do_ work on windows)